examples/move/escrow/api/helpers/create-demo-escrows.ts

```diff
@@ -0,0 +1,101 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { TransactionBlock } from '@mysten/sui.js/transactions';
+
+import { CONFIG } from '../config';
+import { getActiveAddress, getClient, signAndExecute } from '../sui-utils';
+
+const getOwnedBears = async () => {
```

Comment:
Similar to the point I left elsewhere about making it clear that this app would work with any package, we should let people view all their owned objects and potentially lock any of them, not just the `DemoBear`.

Reply:
The main motivation for all these demo data was to have some data to play with  the indexer + for the API to serve some data easily.

I accidently left the README file on the last PR in the stack #15832